types/oidc-provider/index.d.ts

```diff
@@ -898,6 +898,8 @@ export interface Configuration {
 
     formats?: {
         bitsOfOpaqueRandomness?: number | ((ctx: KoaContextWithOIDC, model: BaseModel) => number) | undefined;
+        AccessToken?: (ctx: KoaContextWithOIDC, token: AccessToken) => TokenFormat;
+        ClientCredentials?: (ctx: KoaContextWithOIDC, token: AccessToken) => TokenFormat;
```

Comment:
This is no longer part of the public API and should not be used, hence its inclusion in the types is not welcome. Access Token formats are driven using the `resourceIndicators` configuration options.